### DISCUSSION TITLE
Idea: Immediate Collape of Tree

### DISCUSSION THREAD

**Discussion by egyas** [TriliumNext/Trilium#4883] [Ideas] [answered]
I have a LOT of notes in my notebook, and most of them have several layers of child notes under them.  When I'm on a child note that's 4 layers deep and I get done, then just switch to another note (or have to drill down to find it in the hierarchy), it leaves that first "tree" hanging open for some time.  

I know that it will collapse "after some time", but is there a way to enable IMMEDIATE collapse when you move to a new tree?

If that option exists, I cannot see top find it anywhere.

**Comment by meichthys** [comment]
There is no setting to change this, but you can do `alt-c` to collapse entire tree then `ctrl-.`to expand the tree to the active note.

**Reply by meichthys** [reply, marked as answer]
The shortcuts correspond to these two buttons in the tree: 
![image](https://github.com/user-attachments/assets/ebe6ae91-d116-4d4a-814f-a98ea12a722d)

**Reply by egyas** [reply]
Well, don't I just feel like a stooge.  lol

Sorry for missing that, and thanks for the answer!  :)

**Reply by meichthys** [reply]
no problem! I'm glad you've chosen the open-source option - Trilium!
